log/log.go

```diff
@@ -7,6 +7,60 @@ import (
 	"time"
 )
 
+// Logger ...
+type Logger interface {
+	Infof(format string, v ...interface{})
+	Warnf(format string, v ...interface{})
+	Printf(format string, v ...interface{})
+	Donef(format string, v ...interface{})
+	Debugf(format string, v ...interface{})
+	Errorf(format string, v ...interface{})
+	Println()
+}
+
+type defaultLogger struct {
+}
+
+// NewDefaultLogger ...
+func NewDefaultLogger() Logger {
```

Comment:
Here we used the "default", see my comments above

Reply:
Should we replace it with something more meaningful that adds more context, e.g. `NewCommandLineLogger`?

I can imagine a `Logger` can have multiple implementations in the future (e.g. something that logs to an arbitrary data collector/processor service) so in that sense `Default` would be too ambiguous.

Reply:
Agree with a more specific name is preferable.